cloud/jenkins/pxc-operator-gke-alpha.yml

```diff
@@ -0,0 +1,48 @@
+- job:
```

Comment:
gke jobs are called `alpha`, but openshift `latest`- I propose for these jobs to use `latest` for all so it's consistent

Reply:
https://github.com/Percona-Lab/jenkins-pipelines/pull/1009/commits/5e753a5a785f2f85ea63895d0d7eccf57bfcd61f